core/arch/arm/plat-rdaspen/rdaspen_core_pos.S

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (c) 2025, Arm Limited
+ */
+
+#include <asm.S>
+#include <arm.h>
+#include "platform_config.h"
+
+FUNC get_core_pos_mpidr , :
```

Comment:
Is this function different from `get_core_pos_mpidr()` with `CFG_CORE_CLUSTER_SHIFT=RDASPEN_MAX_CPUS_PER_CLUSTER` and `CFG_CORE_THREAD_SHIFT=RDASPEN_MAX_PE_PER_CPU`?